src/modules/UI/scenes/TransactionList/TransactionList.ui.js

```diff
@@ -265,77 +179,71 @@ export class TransactionList extends Component<Props, State> {
-                        <T style={[styles.request]}>{displayDenomination.symbol}</T>
-                      )}
-                    </View>
-                    <View style={[styles.currentBalanceBoxBitsWrap]}>
-                      <View style={{ flexDirection: 'row' }}>
-                        {displayDenomination.symbol ? (
-                          <T numberOfLines={1} style={[styles.currentBalanceBoxBits, styles.symbol]}>
-                            {displayDenomination.symbol + ' '}
-                            <T numberOfLines={1}>{cryptoAmountString}</T>
-                          </T>
-                        ) : (
-                          <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
-                            {cryptoAmountString}
-                          </T>
-                        )}
+          <View style={styles.balanceBoxContents}>
+            {!isAccountBalanceVisible ? (
+              <View style={[styles.totalBalanceWrap]}>
+                <View style={[styles.hiddenBalanceBoxDollarsWrap]}>
+                  <T style={[styles.currentBalanceBoxHiddenText]}>{SHOW_BALANCE_TEXT}</T>
+                </View>
+              </View>
+            ) : (
+              <View style={styles.balanceShownContainer}>
+                <View style={[styles.iconWrap]}>
+                  {logo ? (
+                    <Image style={[{ height: '100%' }]} source={{ uri: logo }} resizeMode={'cover'} />
+                  ) : (
+                    <T style={[styles.request]}>{displayDenomination.symbol}</T>
+                  )}
+                </View>
+                <View style={[styles.currentBalanceBoxBitsWrap]}>
+                  <View style={{ flexDirection: 'row' }}>
+                    {displayDenomination.symbol ? (
+                      <T numberOfLines={1} style={[styles.currentBalanceBoxBits, styles.symbol]}>
+                        {displayDenomination.symbol + ' '}
+                        <T numberOfLines={1}>{cryptoAmountString}</T>
+                      </T>
+                    ) : (
+                      <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
+                        {cryptoAmountString}
+                      </T>
+                    )}
 
-                        {!displayDenomination.symbol && (
-                          <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
-                            {' ' + selectedCurrencyCode}
-                          </T>
-                        )}
-                      </View>
-                    </View>
-                    <View style={[styles.currentBalanceBoxDollarsWrap]}>
-                      <T numberOfLines={1} style={[styles.currentBalanceBoxDollars]}>
-                        {fiatBalanceString}
+                    {!displayDenomination.symbol && (
+                      <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
+                        {' ' + selectedCurrencyCode}
```

Comment:
what does this change do?

Reply:
I honestly don't even see an actual change. Maybe indentation?